modules/checkout/commerce_checkout.module

```diff
@@ -34,6 +34,13 @@ function commerce_checkout_theme() {
     'commerce_checkout_form__with_sidebar' => [
       'base hook' => 'commerce_checkout_form',
     ],
+    'commerce_checkout_order_summary' => [
+      'variables' => [
+        'order' => NULL,
+        'checkout_flow_plugin_id' => '',
```

Comment:
These are here so that people can do conditionals in the Twig template. For instance, we use this to check if on "order_information" and display "+ applicable taxes" if there are no adjustments. Should we document that somehow here, or in the Twig template?